h2o-algos/src/main/java/hex/schemas/ModelSelectionV3.java

```diff
@@ -114,6 +115,11 @@ public static final class ModelSelectionParametersV3 extends ModelParametersSche
                 level = API.Level.critical)
         public boolean lambda_search;
 
+        @API(help = "For maxrsweep only.  If enabled will attempt to perform sweeping action using multiple nodes in" +
```

Comment:
```suggestion
        @API(help = "For maxrsweep only.  If enabled will attempt to perform sweeping action using multiple nodes in " +
```
missing space after `in` causes `inthe` in R and python doc-strings